### DISCUSSION TITLE
Probelm with the antivirus

### DISCUSSION THREAD

**Discussion by cclowny** [TomSchimansky/CustomTkinter#258] [General]
Hi guys, I few days ago I posted a comment about my antivirus that found my software dangerous (a trojan) since I started using customtkinter.
Is possible that this happens because custom tkinter has a tracker for the system default apparance mode?

**Comment by felipetesc** [comment]
Maybe, and we should look closer to the only CustomTkinter depedency, which is darkdetect. Maybe some changes can be made so darkdetect is an optional requirement? 
Tests can be made:
1) Test A
- uninstall customtkinter from pip
- download darkdetect
- execute a sample of darkdetect
- check if the problem remaims
2) Test B
- uninstall customtkinter from pip
- clone customtkinter
- download darkdetect source code
- build darkdetect and install
- check if the problem remaims
